lib/checkoutUtils.js

```diff
@@ -1,156 +1,60 @@
 import { AsyncStorage } from 'react-native';
-import BASE from '../lib/common';
-
-// TODO get rid of categories (deprecated)
-export const categories = [
-  'All',
-  'Cut Fruit & Packaged Products',
-  'Fruit',
-  'Vegetables',
-  'Frozen & Dried'
-];
-
-// Creates a dictionary object from each product.
+import {
+  createLineItems,
+  createTransactions,
+  getAllProductss,
+  updateCustomers,
+  updateTransactions
+} from './airtable/request';
+
+// Adds a field 'imageUrl' to products
 function createProductData(record) {
-  record = record.fields;
-  return {
-    name: record['Name'],
-    id: record.id,
```

Comment:
Oh is it that other fields like "name" and "points" are already accessibly by dot notation? :0

Reply:
Mhmm! One of the really nice things Aivant implemented with the generator functions is record transformations - ex. "Points Earned" --> `.pointsEarned`; and linked fields will automatically have a '.id' appended to them (so when updating lineItems, the attribute name is `productsPurchasedId`

you can see the object attributes in `schema.js` if you want to sanity-check 😂, but basically using `...record` says "keep everything already there" and then add on the new attributes

Can explain more in person!!